static/.well-known/matrix/client

```diff
@@ -7,5 +7,11 @@
     },
     "org.matrix.msc3575.proxy": {
         "url": "https://slidingsync.lab.matrix.org"
-    }
+    },
+    "org.matrix.rtc_foci": [
```

Comment:
This should be in the format `org.matrix.msc0000.rtc_foci`, using a real MSC number. Our namespace shouldn't be used at a root level like this.

Reply:
@toger5 ^^